presidio-image-anonymizer/setup.py

```diff
@@ -4,6 +4,11 @@
 
 from setuptools import setup, find_packages
 
+requirements = ["flask==1.1.2", "pillow", "pytesseract==0.3.7",
```

Comment:
flask is not a requirement for the package

Reply:
Removed.